man/systemd.netdev.xml

```diff
@@ -1979,6 +1982,77 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Batadv] Section Options</title>
+    <para>The [Batadv] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either "off", "server" or "client"</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>BridgeLoopAvoidance=</varname></term>
+        <listitem>
+          <para>Enables (1) or disables (0) BridgeLoopAvoidance</para>
```

Comment:
This description needs to actually say what the option does... repeating the option name is not enough. (This is not the kernel :]).  Something following this pattern:
> Takes a boolean value. Enables or disables avoidance of bridges with loops with loops. Defaults to true.